src/Sylius/Bundle/ShopBundle/Resources/views/Account/requestPasswordReset.html.twig

```diff
@@ -17,12 +17,14 @@
             </h1>
             <div class="ui segment">
                 {{ form_start(form, {'action': path('sylius_shop_request_password_reset_token'), 'attr': {'class': 'ui large loadable form', 'novalidate': 'novalidate'}}) }}
-                {{ form_row(form.email) }}
+                {{ form_row(form.email, sylius_data_test_attribute_array('reset-email')) }}
 
                 {{ sonata_block_render_event('sylius.shop.request_password_reset_token.form', {'form': form}) }}
 
                 {{ form_row(form._token) }}
-                <button type="submit" class="ui fluid large primary submit button">{{ 'sylius.ui.reset'|trans }}</button>
+                <button type="submit" class="ui fluid large primary submit button" {{ sylius_data_test_attribute_name('request-password-reset-button') }}>
```

Comment:
```suggestion
                <button type="submit" class="ui fluid large primary submit button" {{ sylius_test_html_attribute('request-password-reset-button') }}>
```

Going further with the same logic as in the comment above, this will mark it as specific for HTML and also make the function name shorter.